mappings/net/minecraft/entity/passive/FoxEntity.mapping

```diff
@@ -103,6 +103,7 @@ CLASS net/minecraft/class_4019 net/minecraft/entity/passive/FoxEntity
 	CLASS class_4029 AttackGoal
 		METHOD <init> (Lnet/minecraft/class_4019;DZ)V
 			ARG 2 speed
+			ARG 4 pauseWhenMobIdle
```

Comment:
Given that this is an inner class, we can probably abbreviate the parameter to `pauseWhenIdle`.